src/workspace-data/import-jobs.test.ts

```diff
@@ -96,17 +96,28 @@ describe('useImportJobs', () => {
       canCompute: true,
     };
 
-    it('returns empty list of jobs', () => {
+    it('returns list of jobs', () => {
```

Comment:
The GCP version of this has a better test for "returns list of running jobs in workspace".

Preferably, this would assert that it's only returning jobs for the given workspace, not necessarily all jobs in the store.